assets/xml/objects/object_oyu.xml

```diff
@@ -4,6 +4,7 @@
         <DList Name="object_oyu_DL_000158" Offset="0x158" />
         <Texture Name="object_oyu_Tex_000160" OutName="tex_000160" Format="rgba16" Width="32" Height="32" Offset="0x160" />
         <TextureAnimation Name="object_oyu_Matanimheader_000968" Offset="0x968" />
-        <!-- <Blob Name="object_oyu_Blob_000970" Size="0x50" Offset="0x970" /> -->
+        <!-- freezes ZAPD for some reason <Collision Name="object_oyu_Colheader_000988" Offset="0x988" /> --> 
+        <Blob Name="object_oyu_Blob_000988" Size="0x36" Offset="0x988" />
```

Comment:
Just want to note this is addressed in a ZAPD PR. We can either wait for that, or add a TODO here.

Reply:
Should be addressed with ZAPD update.